packages/components/src/components/form-submit-button/form-submit-button.scss

```diff
@@ -0,0 +1,35 @@
+.dc-form-submit-button {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    align-self: flex-end;
+    padding-top: 2.4em;
```

Comment:
I think w should use `rem` instead. Same with other lines below in the same file.